bin/reth/src/db/mod.rs

```diff
@@ -170,9 +173,18 @@ impl Command {
                                         return Ok(());
                                     }
 
-                                    tui::DbListTUI::<_, tables::$table>::new(|start, count| {
-                                        tool.list::<tables::$table>(start, count).unwrap()
-                                    }, $start, $len, total_entries).run()
+                                    if args.json {
+                                        let list_result = tool.list::<tables::$table>(args.start, args.len)?;
+                                        for (k, v) in list_result {
+                                            let json = serde_json::to_string(&(format!("{:?}", k), v))?;
+                                            println!("{json}");
+                                        }
+                                        Ok(())
```

Comment:
It would make sense, but if we were to do this, we would need to make all the key values implement Serialize. The current TUI implementation uses the fact that they all support Debug.

Would you be happy for this to happen?

Reply:
Yeah I don't mind that.